### PR TITLE
max width out metadata list

### PR DESCRIPTION
**Pull request by abhinandan13jan**
## Fixes 

https://issues.redhat.com/browse/HAC-2920

## Description

setting max-width for metadatDataList labels and text-overflow elipsis

## Type of change

- [ ] Feature
- [x] Bugfix
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## Screen shots / Gifs for design review 

Screenshot 2023-03-03 at 4 33 56 PM
Screenshot 2023-03-03 at 4 34 06 PM
Screenshot 2023-03-03 at 4 34 16 PM

## Browser conformance: 

- [ ] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### CODE REVIEW

**Comment by christianvogt** [comment]
@abhinandan13jan i don't see the ellipsis working with text-overflow in your screenshot

**Comment by abhinandan13jan** [comment]
> @abhinandan13jan i don't see the ellipsis working with text-overflow in your screenshot
@christianvogt 
Updated.. looks like `text-overflow` doesn't work with Label instead wrapped the content in `Truncate` with max-width

File: src/components/PipelineRunDetailsView/MetadataList.tsx [hunk 1]
```diff
@@ -14,8 +16,8 @@ const MetadataList: React.FC<MetadataListProps> = ({ metadata }) => {
       <LabelGroup>
         {labelArray.map((label, index) => (
           <FlexItem key={index}>
-            <Label color="blue" isTruncated={label.toString().length > 100}>
-              {label.toString().replace(',', '=')}{' '}
+            <Label color="blue" className="metadata-list__label">
```
Comment (christianvogt) [inline src/components/PipelineRunDetailsView/MetadataList.tsx:19]: Is this still necessary `className="metadata-list__label"`?
Seems like the PF component should take care of it.

Comment (abhinandan13jan) [reply src/components/PipelineRunDetailsView/MetadataList.tsx:19]: max-width doesn't work without this

Comment (christianvogt) [reply src/components/PipelineRunDetailsView/MetadataList.tsx:19]: Max width doesn't seem necessary and in fact can have negative results by prematurely limiting the width when there is enough space in the column available and 35vw is smaller.

Comment (abhinandan13jan) [reply src/components/PipelineRunDetailsView/MetadataList.tsx]: updated
Screenshot 2023-03-03 at 4 33 56 PM
Screenshot 2023-03-03 at 4 34 06 PM

**Pull request merged**